saige_assoc.py

```diff
@@ -91,28 +102,31 @@ def build_fit_null_command(
         --isCovariateTransform={is_cov_transform} \
         --skipModelFitting={skip_model_fitting} \
         --tol={tol} \
-        --plinkFile={plink_path} \
+        --plinkFile={plink_prefix} \
         --IsOverwriteVarianceRatioFile={is_overwrite_vre_file}
     """
 
 
 # Run single variant association (step 2)
 def build_run_single_variant_test_command(
+    output_path: str,
     vcf_file: str,
-    vcf_file_index: str,
-    vcf_field: str,
-    saige_output_file: str,
     chrom: str,
     cis_window_file: str,
     gmmat_model_path: str,
     variance_ratio_path: str,
+    vcf_field: str = 'GT',
     min_maf: float = 0,
     min_mac: int = 5,
     loco_bool: str = 'FALSE',
     n_markers: int = 10000,
     spa_cutoff: int = 10000,
 ):
-    """Build SAIGE command for running single variant test
+    """
+    n.b. I have not edited this method docstring
+            ... but I have definitely edited the method
```

Comment:
😆

Reply:
ok updated this as well, although I'll have a new PR to deal with all of these parameters in a cleaner way coming soon